install/preview/entrypoint.sh

```diff
@@ -181,6 +181,9 @@ yq e -i '.customCACert.kind = "secret"' config.yaml
 yq e -i '.observability.logLevel = "debug"' config.yaml
 yq e -i '.workspace.runtime.containerdSocket = "/run/k3s/containerd/containerd.sock"' config.yaml
 yq e -i '.workspace.runtime.containerdRuntimeDir = "/var/lib/rancher/k3s/agent/containerd/io.containerd.runtime.v2.task/k8s.io/"' config.yaml
+yq e -i '.workspace.pvc.size = "10Gi"' config.yaml
+yq e -i '.workspace.resources.requests.memory = "500Mi"' config.yaml
```

Comment:
Is this too less? I've opened a couple of workspaces and got a good moving around experience in the browser but I'm sure it could be a bottleneck if the user tries to do more i.e run things, etc.

cc: @corneliusludmann

Reply:
Since we're dealing with memory requests and not limits this should be fine; it's unlikely that a user will be running that many workspaces and will be totally saturating memory. LGTM!

Reply:
Sounds right!